libafl_sugar/src/inprocess.rs

```diff
@@ -158,24 +155,43 @@ where
             let edges_observer = HitcountsMapObserver::new(unsafe {
                 StdMapObserver::from_mut_slice(
                     "edges",
-                    OwnedMutSlice::from_raw_parts_mut(edges_map_mut_ptr(), self.map_size),
+                    OwnedMutSlice::from_raw_parts_mut(
+                        edges_map_mut_ptr(),
+                        EDGES_MAP_ALLOCATED_SIZE,
+                    ),
                 )
             })
             .track_indices();
 
             let cmplog_observer = CmpLogObserver::new("cmplog", true);
 
+            // New maximization map feedback linked to the edges observer and the feedback state
+            let map_feedback = MaxMapFeedback::with_name("map_feedback", &edges_observer);
+            // Extra MapFeedback to deduplicate finds according to the cov map
+            let map_objective = MaxMapFeedback::with_name("map_objective", &edges_observer);
+
+            let calibration = CalibrationStage::new(&map_feedback);
+
+            let stats_stage = AflStatsStage::builder()
+                .map_observer(&edges_observer)
+                .build()?;
+            let stats_stage_cmplog = AflStatsStage::builder()
```

Comment:
why 2nd stage with the same thing?

Reply:
Can't figure out how else to make the type checker happy, the stages seem to have a different type